examples/images_contours_and_fields/contour_label_demo.py

```diff
@@ -57,33 +58,34 @@ def __repr__(self):
     fmt = r'%r \%%'
 else:
     fmt = '%r %%'
-plt.clabel(CS, CS.levels, inline=True, fmt=fmt, fontsize=10)
+
+ax.clabel(CS, CS.levels, inline=True, fmt=fmt, fontsize=10)
 
 ###############################################################################
 # Label contours with arbitrary strings using a dictionary
 
-plt.figure()
+fig1, ax1 = plt.subplots()
 
 # Basic contour plot
-CS = plt.contour(X, Y, Z)
+CS1 = ax1.contour(X, Y, Z)
```

Comment:
Why not reuse `CS`?

Reply:
My thought (which is behind e.g., `fig1, ax1` as well) is that I want it to be completely clear to a totally "n00b" (to programming, python,  or matplotlib) that we're starting over and working on something new.